djangoevents/tests/test_schema.py

```diff
@@ -0,0 +1,87 @@
+import avro.schema
+import json
+import pytest
+from io import StringIO
+from ..schema import decode_cls_name, parse_event_schema, validate_event, event_to_schema_path
+from django.test import override_settings
```

Comment:
please one import per line and sort them :D

OR simply import `schema` and use module (i.e. `schema.decode_cls_name`). This also makes it easier to use `mock.patch.object`.